packages/core/lib/testing/testrunner.js

```diff
@@ -49,231 +50,164 @@ function TestRunner(options = {}) {
   this.TEST_TIMEOUT = (options.mocha && options.mocha.timeout) || 300000;
 }
 
-TestRunner.prototype.initialize = function(callback) {
-  var self = this;
-
-  var test_source = new TestSource(self.config);
+TestRunner.prototype.initialize = async function() {
+  debug("initializing");
+  let test_source = new TestSource(this.config);
   this.config.resolver = new TestResolver(
-    self.initial_resolver,
+    this.initial_resolver,
     test_source,
-    self.config.contracts_build_directory
+    this.config.contracts_build_directory
   );
 
-  var afterStateReset = function(err) {
-    if (err) return callback(err);
-
-    fs.readdir(self.config.contracts_build_directory, function(err, files) {
-      if (err) return callback(err);
-
-      files = _.filter(files, function(file) {
-        return path.extname(file) === ".json";
-      });
-
-      async.map(
-        files,
-        function(file, finished) {
-          fs.readFile(
-            path.join(self.config.contracts_build_directory, file),
-            "utf8",
-            finished
-          );
-        },
-        function(err, data) {
-          if (err) return callback(err);
-
-          var contracts = data.map(JSON.parse).map(json => contract(json));
-          var abis = _.flatMap(contracts, "abi");
+  if (this.first_snapshot) {
+    // // Make the initial deployment (full migration).
+    // await this.deploy();
```

Comment:
Was this intended to be left in?

Reply:
See comment above about commented-out code.